plugins/include/sqlx.inc

```diff
@@ -127,149 +153,231 @@ native SQL_QuoteStringFmt(Handle:db, buffer[], buflen, const fmt[], any:...);
  *  made for each query to simplify driver support.
  * The handler should look like:
  *
- * @param failstate - One of the three TQUERY_ defines.
- * @param query - Handle to the query, do not free it.
- * @param error - An error message, if any.
- * @param errnum - An error code, if any.
- * @param data - Data array you passed in.
- * @param size - Size of the data array you passed in.
- * @param queuetime - Amount of gametime that passed while the query was resolving.
+ *  failstate - One of the three TQUERY_ defines.
+ *  query - Handle to the query, do not free it.
+ *  error - An error message, if any.
+ *  errnum - An error code, if any.
+ *  data - Data array you passed in.
+ *  size - Size of the data array you passed in.
+ *  queuetime - Amount of gametime that passed while the query was resolving.
  * 
  * public QueryHandler(failstate, Handle:query, error[], errnum, data[], size, Float:queuetime)
+ * 
+ * @note The handle you pass in is a DB tuple, NOT and active connection!
+ * @note The handle does no need to be freed.
+ * @note This function is not guaranteed to be in another thread (in fact - it's not).
```

Comment:
This note feels odd. Would be nice if you could take a look at this and think of something nicer.